autoload/lsp/signature.vim

```diff
@@ -41,6 +41,25 @@ export def InitOnce()
   hlset([{name: 'LspSigActiveParameter', default: true, linksto: 'LineNr'}])
 enddef
 
+def LspShowSignatureCb(timer: number)
+  call g:LspShowSignature()
+enddef
+
+# Checks if the cursor moved over a trigger character to
+# not break autopair plugins
+def DetectJumpOverTriggerChar(triggerChars: list<string>)
+  var col = col('.')
+  var lineStr = getline('.')
+
+  for ch in triggerChars
+    if ch == ','
+      break
+    elseif lineStr[col - 2] == ch
```

Comment:
Can you retrieve the character from `lineStr[col - 2]` outside the loop and use it here?